src/main.tsx

```diff
@@ -6,6 +6,7 @@ import Layout from './Layout'
 import AvatarStack from './components/AvatarStack'
 import Home from './home'
 import EmojiReactions from './components/EmojiReactions/EmojiReactions'
+import Claims from './components/Claims/Claims'
```

Comment:
Need to rename the folder and file name to `UserClaims`

Reply:
Done https://github.com/ably-labs/realtime-examples/pull/37/commits/b2f0c9cc51aa0f6156b88db5613df8c6f313a9ae